src/vellum/workflows/nodes/displayable/tool_calling_node/composio_service.py

```diff
@@ -0,0 +1,48 @@
+from dataclasses import dataclass
+from typing import List
+
+from composio_client import Composio as ComposioClient
```

Comment:
[nit] I can see this file living in a new `src/vellum/workflows/integrations` module - if the ToolCallingNode use case goes well, we've also talked about creating a first class composio node